.gitignore

```diff
@@ -3,7 +3,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
-
+.vscode
```

Comment:
I think that IDE hidden files should be tagged in specific part
```
# IDE hidden files
.vscode
.idea
etc.
```

Reply:
Okay, i will fix it.

Reply:
Thank you :)

You can look the link below for distributed configuration of tests 
tests https://github.com/pytorch/ignite/blob/6faa6ac1e3a46c79e0dfcfd976439b86329717b0/tests/ignite/metrics/test_accuracy.py#L781

Basically, you have to create full size tensors on every process 
https://github.com/pytorch/ignite/blob/6faa6ac1e3a46c79e0dfcfd976439b86329717b0/tests/ignite/metrics/test_accuracy.py#L749
Then you work with local bunch of tensors wrt to the rank
https://github.com/pytorch/ignite/blob/6faa6ac1e3a46c79e0dfcfd976439b86329717b0/tests/ignite/metrics/test_accuracy.py#L753
Finally you can compare to sequential results 
https://github.com/pytorch/ignite/blob/6faa6ac1e3a46c79e0dfcfd976439b86329717b0/tests/ignite/metrics/test_accuracy.py#L772

By the way, I think you should add first an integration example (meaning with `Engine`, `process_fn` and `data`).

Thank you again and feel free to ask if you need help.

Reply:
@sdesrozis Thank you, I will try it.

Reply:
@ydcjeff @sdesrozis unfortunately, we have to remove those entries in `.gitignore`. Let's keep the same way it was decided from the beginning to use locally `.git/info/exclude` to exclude files like those. Please, keep `.gitignore` unchanged.


@ydcjeff Btw, thanks for the PR !

Reply:
@vfdev-5 Okay, I will remove it then